packages/form-builder/src/common/storage/interface.ts

```diff
@@ -4,5 +4,5 @@ import {Block} from "../../types/block";
 export interface StorageDriver {
     save({blocks, formSettings}: FormData): Promise<void>,
     load(): FormData,
-    preview(blocks: Block[]): Promise<string>,
+    preview(template: string, blocks: Block[]): Promise<string>,
```

Comment:
@jonwaldstein I'm considering updating the signature of the `preview` method to match the `FormData` interface used by `save` and `load`. Also, if `blocks` is a property of `formSettings` then the interface can probably be simplified.

Reply:
Ah yeah, that would make sense to do.

Reply:
Also, I think it would make more sense to change `template` to `templateId` just so it's more clear that we are referring to the ID specifically.

Reply:
Resolving via https://github.com/impress-org/givewp-next-gen/issues/82.